src/pg/sql/getArgSql.js

```diff
@@ -25,7 +25,7 @@ export default function getArgSql(
     throw Error('pg_arg.order_and_group_unsupported in ' + prefix);
   }
 
-  const meta = (key) =>
+  const meta = (key = null) =>
     $group ? getAggMeta(key, $group, options) : getArgMeta(key, options);
```

Comment:
Please remove $group from the call to getAggMeta, and edit that function (in getMeta.ts) to not use it. We will do all group related key generation in this function.